Projects/Feature/SelfStudyFeature/Sources/Moordinator/SelfStudyMoordinator.swift

```diff
@@ -0,0 +1,32 @@
+import BaseFeature
+import DWebKit
+import UIKit
+import Moordinator
+
+final class SelfStudyMoordinator: Moordinator {
+    private let rootVC = UINavigationController()
+
+    var root: Presentable {
+        rootVC
+    }
+
+    func route(to path: RoutePath) -> MoordinatorContributors {
+        guard let path = path.asDotori else { return .none }
+        switch path {
+        case .selfStudy:
+            return coordinateToNotice()
+
+        default:
+            return .none
+        }
+        return .none
+    }
+}
+
+private extension SelfStudyMoordinator {
+    func coordinateToNotice() -> MoordinatorContributors {
```

Comment:
함수명이 의도와 다른거 같아요 coordinateToSelfStudy로 수정하는건 어떨까요?
```suggestion
            return coordinateToSelfStudy()

        default:
            return .none
        }
        return .none
    }
}

private extension SelfStudyMoordinator {
    func coordinateToSelfStudy() -> MoordinatorContributors {
```

Reply:
보일러 플레이트의 폐해입니다